src/Codex.hs

```diff
@@ -48,9 +48,10 @@ import Codex.Project
 replace :: String -> String -> String -> String
 replace a b c = Text.unpack $ Text.replace (Text.pack a) (Text.pack b) (Text.pack c)
 
-md5hash :: String -> String
-md5hash = concatMap (Printf.printf "%02x") . C8.unpack . MD5.hash . C8.pack
-
+sha1hash :: String -> String
+sha1hash = Text.unpack . Text.decodeUtf8 . sha1 . Text.encodeUtf8 . Text.pack
+  where
+    sha1 = BA.convertToBase BA.Base64 . Crypto.hashWith Crypto.SHA1
```

Comment:
Why are we changing this? It's going to require rehashing everything for every user of the code. Is there a problem that this solves?

Reply:
The change was motivated by removing `cryptohash` (deprecated). The decision to switch to sha1 as well was made in a moment of over caffeination: I'll put md5 back.

Reply:
I'd be fine with SHA1, but we shouldn't break old users of the tool. I think we could try SHA1 first, and if it fails, then we can do MD5. If MD5 succeeds, then we carry on, and eventually write a SHA1 hash to the file instead of MD5.

Reply:
That is a good idea, but I'll leave it for a separate pull request.